core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -297,7 +297,7 @@ func (ec *EthConfirmer) CheckConfirmedMissingReceipt(ctx context.Context) (err e
 	var ethTxIDsToUnconfirm []int64
 	for idx, req := range reqs {
 		// Add to Unconfirm array, all tx where error wasn't NonceTooLow.
-		if req.Error == nil || !evmclient.NewSendError(req.Error).IsNonceTooLowError() {
+		if req.Error == nil || !(evmclient.NewSendError(req.Error).IsNonceTooLowError() || evmclient.NewSendError(req.Error).IsTransactionAlreadyMined()) {
```

Comment:
do we want to check for it here as well, since IsNonceTooLowError is being checked?

Reply:
How about simplifying here to avoid the double `New*()`?
```suggestion
		if req.Error != nil {
			if err := evmclient.NewSendError(req.Error); err.IsNonceTooLowError() || err.IsTransactionAlreadyMined() {
				continue
			}
		}
```
(plus drop 302 and unindent 301)

Reply:
Or, maybe a helper method for `err.IsNonceTooLowError() || err.IsTransactionAlreadyMined()` to reuse would be simpler? Not sure what to name it...

Reply:
Maybe `IsTransactionAlreadyMined()` should be doing the abstraction here and check for both errors internally? @jmank88